.github/workflows/build_and_deploy_tags_to_github_packages.yml

```diff
@@ -0,0 +1,30 @@
+name: Build & Deploy To GitHub Packages
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.7
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+
+    - name: Publish to GPR
+      run: |
+        mkdir -p $HOME/.gem
+        touch $HOME/.gem/credentials
+        chmod 0600 $HOME/.gem/credentials
+        printf -- "---\n:github: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+        gem build *.gemspec
```

Comment:
this should probably be using the rake task, like our ruby gems CI documentation describes

Reply:
Sure - I took that syntax from GH Action's "build and push a gem" boilerplate.
Is our documentation in discourse?

Reply:
I think it is bin/rake build. (we don't use rake release)

Reply:
see: https://github.com/venuenext/venuenext_oauth_config/blob/master/.vnops/rubygems/deploy.sh

Reply:
Thanks - I'll take a look.